### ISSUE TITLE
[BUG] Fbx conversion script only giving out one person

### ISSUE THREAD

**Issue by RohaanA** [mkocabas/VIBE#289]
Thanks for your interest in our research!

If you have problems running our code, please include;

1. your operating system and the version: Windows 11 x64 bit
2. your python version, Python 3.7
3. your pytorch version, 1.4.0
4. the stack trace of the error that you see, No error

Specifically, if you have an issue with pyrender or OpenGL setup & installation, please refer to pyrender [docs](https://pyrender.readthedocs.io/en/latest/) or [github issues](https://github.com/mmatl/pyrender/issues).

Hi, I ran the FbX conversion script on the pkl file the model outputted. However, I can only see one guy in the FBx file, while the model had tracked 2-3 people in the video.  Is it a limitation of 1 person?

**Comment by elia-img** [comment]
@RohaanA I think you are supposed to use the `--person_id` flag specifying the `<tracklet id from VIBE output>` maybe ? Although my question is where do I get the tracklet IDs from... I can't seem to be able to open the .pkl file

**Comment by RohaanA** [comment]
> @RohaanA I think you are supposed to use the `--person_id` flag specifying the `<tracklet id from VIBE output>` maybe ? Although my question is where do I get the tracklet IDs from... I can't seem to be able to open the .pkl file

Yup, that is what I did as well. For the IDs you can simply try starting from 0. The tracklet IDs start from 0 and go up depending on how many people are tracked in the video

**Comment by elia-img** [comment]
> > @RohaanA I think you are supposed to use the `--person_id` flag specifying the `<tracklet id from VIBE output>` maybe ? Although my question is where do I get the tracklet IDs from... I can't seem to be able to open the .pkl file
> 
> Yup, that is what I did as well. For the IDs you can simply try starting from 0. The tracklet IDs start from 0 and go up depending on how many people are tracked in the video

Oh that makes a lot of sense. Thanks for the help :)
